mnt/reform2-nitrogen8m/kernel-config

```diff
@@ -0,0 +1,2180 @@
+ALTERA_MBOX n
+ALTERA_MSGDMA n
+ALTERA_STAPL n
+ALTERA_TSE n
+AMD_PHY n
+AMIGA_PARTITION y
+ANDROID y
+ANDROID_BINDERFS y
+ANDROID_BINDER_DEVICES "binder,hwbinder,vndbinder"
+ANDROID_BINDER_IPC y
+ANDROID_BINDER_IPC_SELFTEST n
+APDS9802ALS n
+APPLE_MFI_FASTCHARGE n
+APPLICOM n
+AQUANTIA_PHY n
+ARCH_ACTIONS n
+ARCH_AGILEX n
+ARCH_ALPINE n
+ARCH_BCM2835 n
+ARCH_BCM_IPROC n
+ARCH_BERLIN n
+ARCH_BRCMSTB n
+ARCH_EXYNOS n
+ARCH_HISI n
+ARCH_K3 n
+ARCH_LAYERSCAPE n
+ARCH_LG1K n
+ARCH_MEDIATEK n
+ARCH_MESON n
+ARCH_MMAP_RND_BITS_MAX 24
+ARCH_MVEBU n
+ARCH_QCOM n
+ARCH_RENESAS n
+ARCH_ROCKCHIP n
+ARCH_S32 n
+ARCH_SEATTLE n
+ARCH_SPRD n
+ARCH_STRATIX10 n
+ARCH_SUNXI n
+ARCH_SYNQUACER n
+ARCH_TEGRA n
+ARCH_THUNDER n
+ARCH_THUNDER2 n
+ARCH_UNIPHIER n
+ARCH_VEXPRESS n
+ARCH_XGENE n
+ARCH_ZX n
+ARCH_ZYNQMP n
+ARCNET n
+ARM64_LSE_ATOMICS y
+ARM64_RELOC_TEST n
+ARM64_VA_BITS 39
+ARM64_VA_BITS_39 y
+ARM64_VA_BITS_48 n
+ARMV8_DEPRECATED y
+ARM_CCI_PMU n
+ARM_CCN n
+ARM_DSU_PMU n
+ARM_IMX6Q_CPUFREQ n
+ARM_IMX8M_DDRC_DEVFREQ n
+ARM_IMX_CPUFREQ_DT y
+ARM_MHU n
+ARM_SBSA_WATCHDOG n
+ARM_SCPI_PROTOCOL n
+ARM_SMMU n
+ARM_SMMU_V3 n
+ARM_SP805_WATCHDOG n
+ARM_SPE_PMU n
+ASHMEM y
+AS_HAS_PAC y
+ATA n
+ATALK n
+ATARI_PARTITION y
+ATA_OVER_ETH n
+ATM n
+AUXDISPLAY n
+AX88796B_PHY n
+B43 n
+B43LEGACY n
+BACKLIGHT_ADP8860 n
+BACKLIGHT_ADP8870 n
+BACKLIGHT_ARCXCNN n
+BACKLIGHT_BD6107 n
+BACKLIGHT_GENERIC n
+BACKLIGHT_GPIO y
+BACKLIGHT_LED y
+BACKLIGHT_LM3630A n
+BACKLIGHT_LM3639 n
+BACKLIGHT_LP855X n
+BACKLIGHT_LV5207LP n
+BACKLIGHT_PWM y
+BACKLIGHT_QCOM_WLED n
+BAREUDP n
+BATMAN_ADV n
+BATTERY_BQ27XXX n
+BATTERY_DS2780 n
+BATTERY_DS2781 n
+BATTERY_DS2782 n
+BATTERY_GAUGE_LTC2941 n
+BATTERY_MAX17040 n
+BATTERY_MAX17042 n
+BATTERY_SBS n
+BCACHE n
+BCM7XXX_PHY n
+BCM84881_PHY n
+BCM87XX_PHY n
+BCMA y
+BCMA_DRIVER_PCI n
+BCMA_HOST_PCI n
+BCM_KONA_USB2_PHY n
+BE2ISCSI n
+BEFS_FS n
+BFS_FS n
+BLK_CGROUP_IOCOST n
+BLK_CGROUP_IOLATENCY n
+BLK_DEV_3W_XXXX_RAID n
+BLK_DEV_CRYPTOLOOP n
+BLK_DEV_DM y
+BLK_DEV_DRBD n
+BLK_DEV_IO_TRACE y
+BLK_DEV_MD y
+BLK_DEV_NBD n
+BLK_DEV_NULL_BLK n
+BLK_DEV_NVME y
+BLK_DEV_PCIESSD_MTIP32XX y
+BLK_DEV_RAM n
+BLK_DEV_RBD n
+BLK_DEV_RSXX n
+BLK_DEV_SKD n
+BLK_DEV_SR n
+BLK_DEV_SX8 n
+BLK_DEV_UMEM n
+BONDING n
+BOOT_PRINTK_DELAY y
+BPF_STREAM_PARSER n
+BRCMFMAC n
+BRCMSMAC n
+BRCMSTB_GISB_ARB n
+BROADCOM_PHY n
+C2PORT n
+CACHEFILES y
+CADENCE_WATCHDOG n
+CAIF n
+CAN n
+CARL9170 n
+CB710_CORE n
+CDROM_PKTCDVD n
+CGROUP_FREEZER y
+CGROUP_HUGETLB n
+CGROUP_RDMA n
+CHARGER_ADP5061 n
+CHARGER_BQ2415X n
+CHARGER_BQ24257 n
+CHARGER_BQ24735 n
+CHARGER_BQ25890 n
+CHARGER_DETECTOR_MAX14656 n
+CHARGER_GPIO n
+CHARGER_LP8727 n
+CHARGER_LT3651 n
+CHARGER_MANAGER n
+CHARGER_MAX8903 n
+CHARGER_RT9455 n
+CHARGER_SBS n
+CHARGER_SMB347 n
+CHARGER_UCS1002 n
+CHECKPOINT_RESTORE y
+CHROME_PLATFORMS n
+CHR_DEV_SCH n
+CHR_DEV_SG y
+CHR_DEV_ST n
+CICADA_PHY n
+CIFS n
+CLEANCACHE n
+CLK_IMX8MM n
+CLK_IMX8MN n
+CLK_IMX8MP n
+CLK_QORIQ n
+CLOCK_THERMAL y
+CMA_SIZE_MBYTES 512
```

Comment:
(ref: this is used when `cma=` is not present on the command-line, and no cma nodes exist in the device tree.)